packages/core/src/Step.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Text from './Text'
+import Icon from 'pcln-icons'
+
+const Container = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Step({ className, isCurrent, isCompleted, children, onClick }) {
+  const color = isCurrent || isCompleted ? 'primary' : 'text.light'
+
+  return (
+    <Container className={className} alignItems="center" onClick={onClick}>
+      {isCompleted && (
+        <Icon name="check" color="primary" size={16} mr={1} legacy={false} />
```

Comment:
Can the `CheckIcon` be imported and used instead?

```jsx
import CheckIcon from 'pcln-icons/lib/Check'
...
<CheckIcon color="primary" size={16} mr={1} />
```

Reply:
I tried this and we ran into a problem with the docs, we looked into everything that we could think of but couldn't resolve it. Using the base icon didn't cause the issue though.